public/js/admin-scripts.js

```diff
@@ -9,3 +9,17 @@ li.each(function(){
 		});
 	};
 });
+
+
+
+ $(document).ready(function () {
+                $("#sidebar").mCustomScrollbar({
+                    theme: "minimal"
+                });
+
+                $('#sidebarCollapse').on('click', function () {
+                    $('#sidebar, #content').toggleClass('active');
+                    $('.collapse.in').toggleClass('in');
+                    $('a[aria-expanded=true]').attr('aria-expanded', 'false');
```

Comment:
You wanna have selectors outside of the event callback. While on small sites the performance hit is negligible , it's still a bad move to have them there. Because on every click, JQ will go through the DOM searching for the selectors.  

Why not just do let sideBarStuff = $('#sidebar, #content')
then
sideBarStuff.toggleClass('active');

Now it will have to search the DOM only once